lib/SPIRV/libSPIRV/SPIRVInstruction.h

```diff
@@ -3707,6 +3684,7 @@ class SPIRVJointMatrixINTELWorkItemInst : public SPIRVJointMatrixINTELInstBase {
   typedef SPIRVInstTemplate<SPIRVJointMatrixINTELWorkItemInst,                 \
                             internal::Op##x##INTEL, __VA_ARGS__>               \
       SPIRV##x##INTEL;
+_SPIRV_OP(JointMatrixWorkItemLength, true, 4)
```

Comment:
This instruction was a part of a legacy spec version, but it's not present in https://github.com/intel/llvm/blob/sycl/sycl/doc/design/spirv-extensions/SPV_INTEL_joint_matrix.asciidoc or https://github.com/intel/llvm/pull/12497
Guess should be removed as well, shouldn't it?